chunky/src/java/se/llbit/chunky/renderer/scene/Sky.java

```diff
@@ -539,6 +587,10 @@ public SkyMode getSkyMode() {
         sky.add("skybox", array);
         break;
       }
+      case SIMULATED: {
+        sky.add("simulatedSky", simulatedSkyMode);
```

Comment:
Should the cache resolution also be read from / written to the scene description file?

Reply:
Implemented in b3fb88d